examples/lighting-app/linux/main.cpp

```diff
@@ -89,8 +89,10 @@ int main(int argc, char * argv[])
         return -1;
     }
 
-    if (LightingMgr().Init() != CHIP_NO_ERROR)
+    CHIP_ERROR err = LightingMgr().Init();
+    if (err != CHIP_NO_ERROR)
     {
+        ChipLogError(AppServer, "Failed to initialize lighting manager: %s", chip::ErrorStr(err));
```

Comment:
nit: prefer CHIP_ERROR_FORMAT (but since this is known linux maybe it does not matter as much)

```cpp
ChipLogError(AppServer, "Failed to initialize lighting manager: %" CHIP_ERROR_FORMAT, err.Format());
```

Reply:
Great advice! I have updated it in the latest commit.